src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -281,6 +281,20 @@ def execution_environment(self, entry: SettingsEntry, config) -> PostProcessorRe
                         )
                     if hint:
                         exit_messages.append(ExitMessage(message=hint, prefix=ExitPrefix.HINT))
+
+            # check for /dev/mqueue when using podman because runner passes ipc=host
+            # https://github.com/ansible/ansible-navigator/issues/610
+            if config.container_engine == "podman" and not Path("/dev/mqueue").exists():
+                exit_msg = (
+                    "Execution environment support while using podman requires a"
+                    " '/dev/mqueue' directory."
+                )
+                exit_messages.append(ExitMessage(message=exit_msg))
+                hint = (
+                    "Try creating it with 'mkdir /dev/mqueue' or reference the documentation for"
```

Comment:
I think it may be reasonable to put this path into a module-level constant
```suggestion
                    "Try creating it with 'mkdir /dev/mqueue/' or reference the documentation for"
```

Reply:
used a var so it's the same everywhere, good idea